examples/hive/sampleapp/README.md

```diff
@@ -1,23 +1,45 @@
 ## Example Application
 
-### On Minikube with Minio 
-0. First configure your minio client `mc`
+### On Minikube with Nooba
+0. You would need an S3 client for this step. We will use `s3cmd` which you can get on MacOS by:
+```
+$ brew install s3cmd 
+```
+and on Linux by using your distribution's package manager. For e.g., on Debian and Ubuntu:
+```
+$ apt-get install s3cmd
+```
+We need to configure `s3cmd` once. So, first, we'll have to gather the connection information:
+```
+$ export S3_HOST=$(minikube service s3 --url | head -n1 | sed -e 's#^http://##') 
+$ export NOOBA_HOME=<path_to_your_Nooba_installation> 
+$ export AWS_ACCESS_KEY_ID=$(${NOOBA_HOME}/noobaa status 2>/dev/null | grep AWS_ACCESS_KEY_ID | awk -F ": " '{print $2}')
+$ export AWS_SECRET_ACCESS_KEY=$(${NOOBA_HOME}/noobaa status 2>/dev/null | grep AWS_SECRET_ACCESS_KEY | awk -F ": " '{print $2}')
+```
+Then, run
+```
+$ s3cmd --configure
+```
+which will open up an interactive session to input various S3 connection parameters stored in the above environment variables. Also, please ensure that you say no to the question about HTTPS connections.
+When all that is done, then check your connection by executing
 
 ```
-$ mc config host rm local 
-$ mc config host add local $(minikube service minio-service --url) minio minio123
+$ s3cmd ls 
 ```
+which should return the buckets created by the example provided in the main installation guide.
+
+To continue this example, 
 
 1. Create a bucket in S3 storage
 
 ```
-$ mc mb local/book-test
+$ s3cmd mb s3://book-test  
 ```
 
 2. Upload the given CSV file to the bucket 
 
 ```
-$ mc cp books.csv local/book-test
+$ s3cmd put books.csv s3://book-test/
```

Comment:
All this could be automated the same way we load example data in NooBaa for testing the regular datasets. There is a docker image wtih _awscli_ that is built during the installation process and you could use that one to load the csv inside the new bucket.

See [data-loader-noobaa.yaml](https://github.com/IBM/dataset-lifecycle-framework/blob/master/examples/noobaa/data-loader-noobaa.yaml) and [noobaa_install.sh](https://github.com/IBM/dataset-lifecycle-framework/blob/master/examples/noobaa/noobaa_install.sh) functions `build_data_loader` and `run_data_loader`.

Reply:
Provided a convenience script for creating the sample application: https://github.com/srikumar003/dataset-lifecycle-framework/commit/ae9a885eed1c985fabbad5155531a2ea7734cb03